src/polyfem/autogen/CMakeLists.txt

```diff
@@ -1,45 +1,53 @@
+set(N_BASES
+	p_n_bases.hpp
+	p_n_bases.cpp
+)
+
+add_library(polyfem_n_bases ${N_BASES})
+add_library(polyfem::n_bases ALIAS polyfem_n_bases)
+target_link_libraries(polyfem_n_bases PRIVATE Eigen3::Eigen)
+
+# -----------------------------------------------------------------------------
+
 set(AUTOGEN_BASES
 	auto_p_bases.cpp
 	auto_p_bases.hpp
 	auto_q_bases.cpp
 	auto_q_bases.hpp
 )
 
+polyfem_set_source_group(${AUTOGEN_BASES})
+if(WIN32)
+	SET_SOURCE_FILES_PROPERTIES(${AUTOGEN_BASES} PROPERTIES COMPILE_FLAGS -Od)
```

Comment:
why?

Reply:
I only rearranged the lines to be more blocked out. I had some compiler issue at some point with this file but a clean build fixed it. I will revert.